### ISSUE TITLE
Bug: Stuck on looting treasure

### ISSUE THREAD

**Issue by Flavescent** [ffxivcode/AutoDuty#9]
On occasion, about once every two runs, AutoDuty gets stuck after looting a treasure chest.  The character is just standing there staring at the disappeared chest, while "Step: Looting Treasure" is displayed. As far it can tell, this bug occurs randomly. The hangup can be cured manually by moving the character away a bit, stopping and restarting the path.

**Comment by steamPUNK70** [comment]
I have the exact same problem o/

**Comment by Zylph** [comment]
I was able to recreate this.  I had "Loot Treasure Coffers" set to "AutoDuty" as the method.  I also already had pandora set to auto open instance chests. I then set the method  to "Pandora" and the issue resolves itself.  My best guess is that pandora is looting the chest before AutoDuty can, and AutoDuty gets stuck attempting to do so. If you stop and restart it, it does not see a coffer, so it moves onto next step.  Maybe AutoDuty can check Pandora's config file and if "Automatically Open Chests" is enabled, then Auto Duty's option for "Method" can be greyed out and forced  to "Pandora" ?   Just giving info, Hope this helps!
